CIScripts/Tests/KSyncTest.ps1

```diff
@@ -0,0 +1,33 @@
+function Test-KSync {
```

Comment:
I think it would be better to create something like `Test-Agent` function, since there will be more Agent tests in the future. There could be a list of executables and a loop running them all.